flow-typed/fusion-core.js

```diff
@@ -20,49 +20,36 @@ declare module 'fusion-core' {
     },
   } & KoaContext;
   declare export type Context = SSRContext | KoaContext;
-  declare export type FusionPlugin<
-    Dependencies,
-    Service
-  > = Dependencies => Service;
+  declare export type FusionPlugin<Deps, Service> = {
+    deps?: Deps,
```

Comment:
I prefer `requires` over `deps`

Reply:
And here the bikeshed begins lol. I'm kinda okay with either, I keep going back and forth.

Anyone else have thoughts?